### PR TITLE
adding ECCN element

### PR DESCRIPTION
**Pull request by jkrech**
PR for issue #334 adding ECCN

### CODE REVIEW

**Comment by jkrech** [comment]
Dates are set for merge on 2024-11-12 therefore validation is expected to fail today.

**Review by jkrech — COMMENTED** [review]
LGTM

File: schema/PACK.xsd [hunk 1]
```diff
@@ -18,13 +18,14 @@
...
 
-  $Date:        10. Oct 2024
-  $Revision:    1.7.42
+  $Date:        12. Nov 2024
+  $Revision:    1.7.43
   $Project: Schema File for Package Description File Format Specification
 
-  Package file name convention <vendor>.<name>.<version>.pack
-  SchemaVersion=1.7.42
+  Package file name convention <vendor>.<name>.<version>.pack SchemaVersion=1.7.43
```
Comment (brondani) [inline schema/PACK.xsd:25]: Was the newline removal intentional?
```suggestion
  Package file name convention <vendor>.<name>.<version>.pack
  SchemaVersion=1.7.43
```

**Review by OCHSTM — APPROVED** [review]
Looks good to me

**Pull request merged**